src/Jupyter/Visualization/StateDisplayEncoders.cs

```diff
@@ -15,6 +15,15 @@
 
 namespace Microsoft.Quantum.IQSharp.Jupyter
 {
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum PhaseDisplayStyle
+    {
+        None,
```

Comment:
Is having a `None` option something that is useful here? I think it may just leave empty cells in the table. Is there a reason someone might prefer that?

Reply:
I think giving people an option to not show phase at all might be helpful to keep. Also, in case I want to add more features to show phase, it might be helpful to have an option to show nothing/

Reply:
That makes sense!